model/k8s/metrics.yaml

```diff
@@ -63,6 +63,19 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
+    type: metric
+    metric_name: k8s.container.status.state
+    stability: experimental
+    brief: ""
+    instrument: updowncounter
+    unit: ""
```

Comment:
Based on the discussion from https://github.com/open-telemetry/semantic-conventions/pull/1668#discussion_r1907556971 I think that should be `{container}`?